### PR TITLE
[UI] Create ui to register new user#11

### PR DESCRIPTION
**Pull request by PaulGrishunin**
Add form validations
Add redirect to Main if status.ok

### CODE REVIEW

File: RegistrationView.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        type="text" pattern="[a-zA-Z0-9]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]+"
+        title="ENTER A VALID Email ADDRESS. example@example.com"
+        placeholder="E-mail"
+        value={this.state.email}
+        onChange={this.handleEmailChange}
+        required
+        />
+        <input
+        type="password" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{6,}"
+        title="Your password must be at least 6 characters as well as contain at least one uppercase, one lowercase, and one number."
```
Comment (SzymonSmykala) [inline RegistrationView.js:95]: I think it is 5 characters on a backend side

Comment (ZPoreba) [reply RegistrationView.js:95]: Do we need double validation on backend and frontend ?

**Comment by ZPoreba** [comment]
We run frontend from directory /client so plaese do not put any file in different directory.

Frontend -> /client
Backend -> /server

File: RegistrationView.js (hunk 1 above)
Comment (SzymonSmykala) [reply RegistrationView.js:95]: yes, make UX the best

Comment (PaulGrishunin) [reply RegistrationView.js:95]: Na dobre wadidacja powinna być zarówno na frontendzie, jak i na bekendzie. Na froncie pierwotne przesiewanie abrakadabry, którą użytkownik może wysłać.

**Comment by PaulGrishunin** [comment]
@ZPoreba   I do not understand how it happens that the files are not sent to the directories in which they should be. For the 2nd time the same thing. Apparently I am trying incorrectly to update files via Git.      
In the routes.js file, I added a line   <Route path="/signedup" />   , so I update it. 
 How can I move these files to the right places now?

**Comment by SzymonSmykala** [comment]
> @ZPoreba I do not understand how it happens that the files are not sent to the directories in which they should be. For the 2nd time the same thing. Apparently I am trying incorrectly to update files via Git.
> In the routes.js file, I added a line , so I update it.
> How can I move these files to the right places now?

just move them under /client folder

**Pull request merged**